superduper/base/apply.py

```diff
@@ -310,11 +310,11 @@ def wrapper(child):
 
     if not these_job_events:
         if not CFG.json_native:
-            metadata_event.data['status'] = json.dumps(ready_status())
+            metadata_event.data['status'] = json.dumps(running_status())
         else:
-            metadata_event.data['status'] = ready_status()
+            metadata_event.data['status'] = running_status()
```

Comment:
These should be `ready_status()`

Reply:
I 've deprecated ready for running. This is to have consistent semantics across the framework.

Reply:
If that’s the case, then it should be a success status, since the status set here indicates the component has already finished.